packages/contracts-bedrock/contracts/universal/Proxy.sol

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+/**
+ * @title Proxy
+ * @dev Basic transparent proxy contract
+ */
+contract Proxy {
+    /*************
+     * Constants *
+     *************/
+
+    // bytes32(uint256(keccak256('eip1967.proxy.implementation')) - 1)
+    bytes32 internal constant IMPLEMENTATION_KEY =
+        0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc;
+
+    // bytes32(uint256(keccak256('eip1967.proxy.admin')) - 1)
+    bytes32 internal constant OWNER_KEY =
+        0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103;
+
+    /***************
+     * Constructor *
+     ***************/
+
+    /**
+     * @param _owner Address of the initial contract owner.
+     */
+    constructor(address _owner) {
+        _setOwner(_owner);
+    }
+
+    // slither-disable-next-line locked-ether
+    fallback() external payable {
+        // Proxy call by default.
+        _doProxyCall();
+    }
+
+    modifier onlyOwner() {
+        address owner = _getOwner();
+        require(msg.sender == owner);
+        _;
+    }
+
+    /**********************
+     * External Functions *
+     **********************/
+
+    /**
+     * @param _implementation The address of the implementation contract
+     */
+    function setImplementation(address _implementation) external onlyOwner {
+        _setImplementation(_implementation);
+    }
+
+    /**
+     * Changes the owner of the proxy contract. Only callable by the owner.
+     * @param _owner New owner of the proxy contract.
+     */
+    // slither-disable-next-line external-function
```

Comment:
This disable comment is unnecessary, it enforces [this](https://github.com/crytic/slither/wiki/Detector-Documentation#description-108):
> public functions that are never called by the contract should be declared external to save gas.

Reply:
similar for the other occurences of it.